Src/Base/AMReX_TinyProfiler.H

```diff
@@ -23,13 +23,16 @@ class TinyProfiler
 {
 public:
     explicit TinyProfiler (std::string funcname) noexcept;
-    TinyProfiler (std::string funcname, bool start_) noexcept;
+    TinyProfiler (std::string funcname, bool start_, bool useCUPTI) noexcept;
```

Comment:
Could we change this to `TinyProfiler (std::string funcname, bool start_, bool useCUPTI=false) noexcept;`?  And the `const char*` version?

Reply:
Done.